Gruntfile.coffee

```diff
@@ -740,19 +726,18 @@ module.exports = (grunt) ->
 				tasks: [
 					"build"
 				]
-			lib_test:
-				files: "<%= jshint.lib_test.src %>"
-				tasks: [
-					"jshint:lib_test"
-				]
-			source:
-				files: "<%= jshint.lib_test.src %>"
-				tasks: [
-					"build"
+			js:
+				files: "<%= eslint.all.src %>"
+				tasks: "js"
+
+		eslint:
+			options:
+				configFile: if process.env.CI == "true" then ".eslintrc.ci.json" else ".eslintrc.json"
```

Comment:
@nschonni @LaurentGoderre Any thoughts on how best to go about referencing ESLint's settings?

Currently this line is referencing copies of wet-boew/wet-boew's .eslintrc.ci.json and .eslintrc.json files. But an alternate way would be to not include copies of those files in this PR and just reference the originals from the "lib/wet-boew" folder.

It seems like GCWeb and the other theme repos are currently using a mix of both ways:

* JSHint settings were previously being referenced from "lib/wet-boew/.jshintrc". But since that file was removed at some point from wet-boew/wet-boew, it's been over a year since the last time GCWeb was able to actually inherit those settings.
* Sass-lint is setup to use a copy of .sass-lint situated in the GCWeb repo. GCWeb and wet-boew/wet-boew's versions of that file are exactly identical.

Reply:
I'm gonna update this PR to modify this line to point to lib/wet-boew's versions of those files and remove my copies of those files.